input/musig2_session_manager.go

```diff
@@ -134,9 +137,7 @@ func (m *MusigSessionManager) MuSig2CreateSession(bipVersion MuSig2Version,
 	//
 	// We'll use just all zeroes as the session ID for the mutex, as this
 	// is a "global" action.
```

Comment:
This comment is now out-of-date.  I think we should replace it with a sentence explaining why we don't need to acquire `sessionMtx[session.SessionID]` here.